docs/de/getting-started/usage.md

```diff
@@ -7,36 +7,35 @@ Nach dem Start siehst Du den Hauptbildschirm der Community-Firmware mit seinen r
 
 ![About](https://raw.githubusercontent.com/wiki/ftCommunity/ftcommunity-TXT/about.png "Screenshot der 'About'-App")
 
-Einige Apps &mdash; darunter die "About"-App &mdash; stellen zusätzliche Optionen zur Verfügung, die über das "Drei-Balken-Symbol" links in der Titelzeile erreichbar sind:
+Einige Apps (darunter die "About"-App) stellen zusätzliche Optionen zur Verfügung, die über das "**Drei-Balken-Symbol**" links in der Titelzeile erreichbar sind:
 
 ![About](https://raw.githubusercontent.com/wiki/ftCommunity/ftcommunity-TXT/about-with-menu.png "Screenshot  der 'About'-App mit ausgeklapptem Options-Menü")
 
-Mit dem "x" rechts in der Titelzeile oder mit einem kurzen Druck (eine Sekunde oder kürzer) auf den blau leuchtenden "ON/OFF"-Schalter des TXT beendest Du die App und kommst zum Hauptbildschirm zurück.
+Mit dem "**X**" rechts in der Titelzeile oder mit einem kurzen Druck (eine Sekunde oder kürzer) auf den blau leuchtenden "**ON/OFF**"-Schalter des TXT beendest Du die App und kommst zum Hauptbildschirm zurück.
 
-Ein längerer Druck (zwei Sekunden oder mehr) auf den "ON/OFF"-Schalter fährt den TXT herunter und schaltet ihn ab.
+Ein längerer Druck (_zwei Sekunden oder mehr_) auf den "**ON/OFF**"-Schalter fährt den TXT herunter und schaltet ihn ab.
 
 # Verbindung mit einem WLAN
 
-Mit der "WiFi"-App kannst Du den TXT mit einem bestehenden WLAN-Netzwerk verbinden.
+Mit der "**WiFi**"-App kannst Du den TXT mit einem bestehenden WLAN-Netzwerk verbinden.
 
-Die Netzwerkeinstellungen werden auf der SD-Karte gespeichert, und der TXT verbindet sich beim nächsten Start automatisch wieder mit dem Netzwerk, wenn es in Reichweite ist.
+Die Netzwerkeinstellungen werden auf der MicroSD-Karte gespeichert, und der TXT verbindet sich beim nächsten Start automatisch wieder mit dem Netzwerk, **wenn es in Reichweite ist**.
 
-Eine bestehende WLAN-Verbindung wird durch ein kleines WLAN-Signal-Symbol in der Statusleiste des Hauptbildschirms angezeigt.
+(_Eine bestehende WLAN-Verbindung wird durch ein kleines WLAN-Signal-Symbol in der Statusleiste des Hauptbildschirms angezeigt._)
 
 # Zugriff auf das Web-Interface
 
-Die Community-Firmware stellt ein Web-Interface zur Verfügung, mit der man im Browser die auf dem TXT installierten Apps starten und verwalten kann. Um auf das Web-Interface zuzugreifen, gibst Du `http://<ip-addresse>` in die Addresszeile des Browsers ein. Dabei ist `<ip-addresse>` die IP-Adresse Deines TXT. Die IP-Addresse wird angezeigt, wenn Du die Statuszeile des Hauptbildschirms antippst.
+Die Community-Firmware stellt ein Web-Interface zur Verfügung, mit der man im Browser die auf dem TXT installierten Apps starten und verwalten kann. Um auf das Web-Interface zuzugreifen, gibst Du "**http://IP-Adresse**" in die Addresszeile des Browsers ein. Dabei ist "**IP-Adresse**" die IP-Adresse des TXT. Die IP-Addresse wird angezeigt, wenn Du die Statuszeile des Hauptbildschirms antippst.
 
```

Comment:
Die Hervorhebung in `Programmcode-Schrift` war hier beabsichtigt, um anzudeuten dass man die URL bzw. die IP-Adresse irgendwo anders eintippt. Sollte man denke ich nicht ändern.